inlang/source-code/sdk/src/listProjects.ts

```diff
@@ -0,0 +1,37 @@
+import type { NodeishFilesystem } from "@lix-js/fs"
+
+export const listProjects = async (
+	nodeishFs: NodeishFilesystem,
+	from: string,
+	recursionLimit: number = 3
```

Comment:
I propose to remove the recursion limit as an argument. 

We will remove this argument in the future when we have caching. If we expose the argument and then remove it -> breaking change. Hence, I propose to hardcode a number (lets increase it to 5?).

Reply:
@samuelstroschein alright, makes sense